src/util/achievement-data.tsx

```diff
@@ -39,6 +40,12 @@ export enum AchievementId {
   PROFIL_DETEKTIV = "PROFIL_DETEKTIV",
   PRIVATSPHAERE_QUIZ = "PRIVATSPHAERE_QUIZ",
   PRIVATSPHAERE_SWIPE = "PRIVATSPHAERE_SWIPE",
+
+  DATA_PROCESSING_CHECKPOINT_STARTED = "#DATA_PROCESSING_CHECKPOINT_STARTED",
+  DATA_PROCESSING_CHECKPOINT_INTRODUCTION = "#DATA_PROCESSING_CHECKPOINT_INTRODUCTION",
+  DATA_PROCESSING_CHECKPOINT_COLLECT = "#DATA_PROCESSING_CHECKPOINT_COLLECT",
+  DATA_PROCESSING_CHECKPOINT_PROCESS = "#DATA_PROCESSING_CHECKPOINT_PROCESS",
+  DATA_PROCESSING_DONE = "DATA_PROCESSING_DONE",
```

Comment:
ich hatte schon ein achievement dafür erstellt. also entweder du benutzt das oder benennst das was ich erstellt hab um. So oder so würde ich mal main reinmergen

Reply:
Ob wir des WERBE_GURU noch brauchen weiß ich nicht, würd ich nicht in dem PR löschen